kubernetes/helm/pinot/Chart.yaml

```diff
@@ -21,7 +21,7 @@ apiVersion: v1
 appVersion: 0.2.5-SNAPSHOT
```

Comment:
Even though this is supposed to be the application in side this chart's version, does it make sure to match the version field on line 24?

Reply:
I don't know, I asked on Slack.
I'll just make them the same now and change later if I'm asked to.